markdownlint.js

```diff
@@ -78,7 +78,8 @@ function readConfiguration(args) {
 
 function prepareFileList(files, fileExtensions, previousResults) {
   const globOptions = {
-    nodir: true
+    nodir: true,
+    dot: options.dot != null ? true : false
```

Comment:
This line of code could be refactored like this `dot: options.dot ?? false` 
But you're supporting `node >=10`, and the nullish coalescing operator (??), only has been introduced in `node >= 14` according to [node.green](https://node.green/).

Reply:
I think `!!options.dot` is the most conventional way to do this.

Reply:
Also, please sort alphabetically - here and in the list of options in MD and JS.